{ 'cebit_low_stand': { 'head': [],
                       'left_arm': [],
                       'left_leg': [ { 'duration': 2.024,
                                       'name': u'cebit_low_stand',
                                       'positions': [ 0.0101,
                                                      -0.0889,
                                                      -1.1714,
                                                      1.837,
                                                      0.7288,
                                                      -0.0701],
                                       'starttime': 0.0}],
                       'right_arm': [],
                       'right_leg': [ { 'duration': 2.024,
                                        'name': u'cebit_low_stand',
                                        'positions': [ -0.0101,
                                                       0.0889,
                                                       1.1714,
                                                       -1.837,
                                                       -0.7288,
                                                       0.0701],
                                        'starttime': 0.0}],
                       'torso': []}}
